snippets/card-product.liquid

```diff
@@ -200,6 +200,11 @@
             {%- endif -%}
 
             {% render 'price', product: card_product, price_class: '' %}
+            {%- if card_product.quantity_price_breaks_configured? -%}
+              <div class="card__information--volume-pricing-note">
+                <span class="caption">{{ 'products.product.volume_pricing.note' | t }}</span>
```

Comment:
this locales was added in different PR, is it right?

Reply:
Yes I branched off of that :)